scripts/install-worker.sh

```diff
@@ -59,13 +59,18 @@ sudo yum install -y \
     chrony \
     conntrack \
     curl \
-    jq \
     ec2-instance-connect \
+    ipvsadm \
+    jq \
     nfs-utils \
     socat \
     unzip \
     wget \
-    ipvsadm
+    yum-plugin-versionlock
+
+# Downgrade and lock ec2-utils until 1.2-47 is available: https://github.com/aws/amazon-ec2-utils/issues/22
+sudo yum downgrade -y ec2-utils-1.2-45.amzn2.noarch
+sudo yum versionlock ec2-utils-*
```

Comment:
How did we validate this change?

Reply:
I built a `1.21` AMI, created a `t2.micro` with it, and verified that `yum list | grep ec2-utils` showed `1.2-45`.